### ISSUE TITLE
Bad link issues

### ISSUE THREAD

**Issue by oliver408i** [victowang/WikiGame#1]
Example: on https://en.wikipedia.org/wiki/Crossbow (target is Polyethylene terephthalate), the next page is dowel, but the only link with dowel pins, which is not dowel

**Comment by victowang** [comment]
Hi !
Thanks for taking interest in this small project !

It seems to me that the issue is this specific Wikipedia page, not the code. 
Looking at the HTML of the https://en.wikipedia.org/wiki/Crossbow page, I see:
`<a href="/wiki/Dowel_pin" class="mw-redirect" title="Dowel pin">dowel pins</a>`
But when you try to go to https://en.wikipedia.org/wiki/Dowel_pin, you get redirected to https://en.wikipedia.org/wiki/Dowel#In_machinery.
And I did not find an existing Wikipedia page dedicated to "Dowel pins"

I don't see any code changes in this project that would fix this bad link.

**Comment by oliver408i** [comment]
Ah I see, but the path that it returned as dowel, and my program I wrote that uses this tried to look for a link to dowel, not dowel pin. Maybe you can change it so that it appends dowel pins to the path instead of dowel. If you would like to take a look at my project, I can share it :)

**Comment by victowang** [comment]
Right, will try to see if I can fix the displayed path to match what the a real user would see when I have a bit of time for it :)

**Comment by oliver408i** [comment]
Acutally I was able to fix it, but my version is modified to use selenium, so I can't really open a pull request
